src/calva-fmt/src/format.ts

```diff
@@ -71,7 +72,14 @@ export async function formatRangeEdits(
 
 export async function formatRange(document: vscode.TextDocument, range: vscode.Range) {
   const wsEdit: vscode.WorkspaceEdit = new vscode.WorkspaceEdit();
-  wsEdit.set(document.uri, await formatRangeEdits(document, range));
+  const edits = await formatRangeEdits(document, range);
+
+  if (isUndefined(edits)) {
+    console.error('formatRangeEdits returned undefined!', cloneDeep({ document, range }));
+    return false;
+  }
+
+  wsEdit.set(document.uri, edits);
```

Comment:
As discussed, if edits is undefined we log an error and return false instead of trying to apply undefined edits.